client/store/configureStore.js

```diff
@@ -7,7 +7,9 @@ import rootReducer from '../reducers/index';
 import saveTokenMiddleware from '../middleware/auth';
 
 // blacklist ui state so they're not persisted
-const config = { key: 'root', storage, blacklist: ['uiReducers'] };
+const config = {
+  key: 'root', storage, blacklist: ['uiReducers', 'events'],
```

Comment:
Can we remove `events` from the blacklist?

Reply:
@Alameen688 removing `events` from the blacklist persists events in the store. The means events would always be defined. Hence setting a boolean check would not work.

Reply:
It's okay then.